src/includes/getting-started-primer/javascript.nextjs.mdx

```diff
@@ -0,0 +1,14 @@
+<Note>
+
+_Sentry's Next.js SDK enables automatic reporting of errors and exceptions._
```

Comment:
Isn't Performance monitoring included as well?

Reply:
Yes, but I didn't include it in the docs since it isn't finished yet. This will be updated when it's done, and maybe additional pages.